app/export_test.go

```diff
@@ -703,3 +703,40 @@ func TestBuildPostReplies(t *testing.T) {
 		}
 	})
 }
+
+func TestExportDeletedTeams(t *testing.T) {
+	th1 := Setup(t).InitBasic()
+	defer th1.TearDown()
+
+	team1 := th1.CreateTeam()
+	channel1 := th1.CreateChannel(team1)
+	th1.CreatePost(channel1)
+
+	// Delete the team to check that this is handled correctly on import.
+	err := th1.App.SoftDeleteTeam(team1.Id)
+	require.Nil(t, err)
+
+	var b bytes.Buffer
+	err = th1.App.BulkExport(&b, "somePath", BulkExportOpts{})
+	require.Nil(t, err)
+
+	th2 := Setup(t)
+	defer th2.TearDown()
+	err, i := th2.App.BulkImport(th2.Context, &b, nil, false, 5)
+	assert.Nil(t, err)
+	assert.Equal(t, 0, i)
+
+	teams1, err := th1.App.GetAllTeams()
+	assert.Nil(t, err)
+	teams2, err := th2.App.GetAllTeams()
+	assert.Nil(t, err)
+	assert.Equal(t, len(teams1), len(teams2))
```

Comment:
Maybe just do a quick spot check to see that `team1.Id` is not there in the response?